docs/RELEASING.md

```diff
@@ -1,43 +1,53 @@
 Releasing
 =========
 
-The following steps were required to release using R Studio.
+Prepare the release
+-------------------
 
-1. From the R Studio `Build` menu, select `Install and Restart`.
+  The following steps were required to prepare a release using R Studio.
+  1. From the R Studio `Build` menu, select `Install and Restart`.
+  2. From the R Studio `Build` menu, select `Check Package`.
 
-2. From the R Studio `Build` menu, select `Check Package`.
 
+  The following steps were required to prepare a releash to CRAN using a Mac. (Currently using branch: `CRANTryTwo`)
 
-The following steps were required to releash to CRAN using a Mac. (Currently using branch: `CRANTryTwo`)
+  1. Install R.
 
-1. Install R.
+         brew install r
 
-       brew install r
+  2. Install tex tools. Note: Need to close and reopen terminal (and/or RStudio) to see `pdflatex` on the path.
 
-2. Install tex tools. Note: Need to close and reopen terminal (and/or RStudio) to see `pdflatex` on the path.
-
-       brew cask install mactex
+         brew cask install mactex
        
-3. Install pandoc to check .md files.
+  3. Install pandoc to check .md files.
 
-       brew install pandoc
+         brew install pandoc
     
-4. Install [RStudio](https://rstudio.com/products/rstudio/download/#download).
-
-5. Open [oysteR.Rproj](../oysterR.Rproj) in RStudio.
+  4. Install [RStudio](https://rstudio.com/products/rstudio/download/#download).
 
-6. Setup devtools.
+  5. Open [oysteR.Rproj](../oysterR.Rproj) in RStudio.
 
+  6. Setup devtools.
    In R Console tab, run: `install.packages("devtools")`
 
-7. Run R Command to build.
+  7. Run R Command to build.
 
-       R CMD build .
+         R CMD build .
     
-8. Run R Command to check.
+  8. Run R Command to check.
 
-       R CMD check *tar.gz --as-cran
+         R CMD check *tar.gz --as-cran
```

Comment:
While this works, it also caused some pain for you @bhamail , might be worth noting to use the actual file name generated by `R CMD build .`

Reply:
> While this works, it also caused some pain for you @bhamail , might be worth noting to use the actual file name generated by `R CMD build .`

Ah, good point. Fixed here: 3431ad0